src/preferences/dialog/dlgprefcolors.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include <QWidget>
+
+#include "control/controlproxy.h"
+#include "preferences/colorpalettesettings.h"
+#include "preferences/dialog/ui_dlgprefcolorsdlg.h"
+#include "preferences/dlgpreferencepage.h"
+#include "preferences/usersettings.h"
+
+class DlgPrefColors : public DlgPreferencePage, public Ui::DlgPrefColorsDlg {
+    Q_OBJECT
+  public:
+    DlgPrefColors(QWidget* parent, UserSettingsPointer _config);
+    virtual ~DlgPrefColors();
+
+  public slots:
+    // Apply changes to widget
+    void slotApply();
+    void slotResetToDefaults();
+
+  signals:
+    void apply(const QString&);
+
+  private:
+    void loadSettings();
+    void loadPaletteIntoEditor(const ColorPalette& palette);
+
+    // Pointer to config object
```

Comment:
This comment adds no value. All those members that are initialized once in the constructor could also be declared as const.